README.md

```diff
@@ -151,19 +156,20 @@ Example:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-A popup menu for work cards.
+- The more button on the mobile homepage will show the rest of the speakers
+- Add more pages such as Programs, Join, sponsor will be added
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- CONTRIBUTING -->
 
-## 🤝 Contributing <a name="contributing"></a>
+## 🤝 Credit <a name="credit"></a>
 
-Contributions have been made and accepted by:
+Original design idea by Cindy Shin in [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
 
-- GitHub: [BranBayou](https://github.com/BranBayou)
+## 🤝 Contributing <a name="contributing"></a>
 
-- GitHub: [touredri](https://github.com/touredri)
+<!-- Contributions have been made and accepted by: -->
 
 Contributions, issues, and feature requests are welcome!
 
```

Comment:
- [ ] You did a wonderful job with creating the README file :+1:, but kindly don't forget to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). It should be done in the 'Acknowledgements' section